package.json

```diff
@@ -13,7 +13,7 @@
   "author": "Tapio Vierros",
   "dependencies": {
     "chokidar": "^2.0.4",
-    "colors": "latest",
+    "colors": "1.4.0",
     "connect": "^3.6.6",
     "cors": "latest",
```

Comment:
might as well pin the rest of 'latest' deps

Reply:
Yes, you are right! I will fix in a forked version of this project https://www.npmjs.com/package/dead-server because this one seems to be abandoned 😥